compiler-rt/lib/scudo/standalone/combined.h

```diff
@@ -1501,6 +1503,11 @@ class Allocator {
                 getPageSizeCached()),
         "scudo:ring_buffer");
     RawRingBuffer = reinterpret_cast<char *>(MemMap.getBase());
+    if (RawRingBuffer == nullptr) {
+      Printf("Failed to allocate allocation ring buffer of size %d",
+             getFlags()->allocation_ring_buffer_size);
+      return;
```

Comment:
I think this is initialized pretty early. If it fails in the buffer preparation, we may want to abort and see what's wrong here?

Reply:
That's also possible. The main thing I was thinking of `ENOMEM`.

Reply:
Okay, I was thinking the case that if `allocation_ring_buffer_size` is too big, we may want to change a smaller number immediately instead of running the process for a while and then wonder why there's no stack stored. 

No strong opinion here